aopy/data/bmi3d.py

```diff
@@ -380,7 +381,56 @@ def get_ecube_digital_input_times(path, data_dir, ch):
 #####################
 # Preprocessed data #
 #####################
-def get_interp_kinematics(exp_data, datatype='cursor', samplerate=1000):
+def get_target_events(exp_data, exp_metadata):
+    '''
+    For target acquisition tasks, get an (n_event, n_target) array encoding the position
+    of each target. When the target is off, its position is NaN. Can be used to generate a
+    sampled timeseries using :func:`~aopy.data.bmi3d.get_kinematic_segments`
+
```

Comment:
I'm having a hard time understanding what this function does. What is the relevance of the events and what are they? It seems like this goes through and gets the unique position of each target?